source/documentation/deploying_services/use_a_custom_domain.erb

```diff
@@ -1,32 +1,43 @@
 # Managing custom domains using the cdn-route service
 
-This section explains how to manage custom domains for your app with a content distribution network (CDN). The CDN:
+You can run your app on a custom domain instead of the default domain of `NAME.cloudapps.digital`.
 
-- routes requests to your app
-- [caches](#caching) responses
+You should use a subdomain instead of an apex domain. For example, you should use `www.example.com` instead of `example.com`.
 
-You can use the GOV.UK PaaS cdn-route service to set up the CDN.
+Once you have chosen a domain, you must:
 
-With the CDN in place, your app will be available on a domain of your choice through HTTPS. The GOV.UK PaaS redirects non-secure HTTP requests to a secure version of your app.
+- register that domain with your organisation's preferred domain registrar
+- use your organisation's preferred domain name system (DNS) provider to host DNS
+- manage the domain's TLS certificates
+- route requests to GOV.UK PaaS
+- cache responses (optional)
 
-You must configure the cdn-route service to use a subdomain. If you configure the service to use an apex domain, the service creation might not succeed. The following table contains examples of apex domains and subdomains:
+GOV.UK PaaS does not provide domain registration or DNS hosting. Use your organisation's preferred registrar and DNS provider.
 
-<div style="height:1px;font-size:1px;">&nbsp;</div>
+You can use either the GOV.UK PaaS [cdn-route service](https://github.com/alphagov/paas-cdn-broker) or use a commercial content distribution network (CDN) to manage TLS and route requests to GOV.UK PaaS.
```

Comment:
General feedback and not something we need to fix: It's pleasantly transparent to link to our https://github.com/alphagov/paas-cdn-broker repo, but I don't think its README is very tenant-friendly.